src/App/Pages/Send/SendsPageViewModel.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Bit.App.Utilities;
+using Bit.Core.Abstractions;
+using Bit.Core.Models.View;
+using Bit.Core.Utilities;
+using Xamarin.Forms;
+
+namespace Bit.App.Pages
+{
+    public class SendsPageViewModel : BaseViewModel
+    {
+        private readonly ISearchService _searchService;
+
+        private CancellationTokenSource _searchCancellationTokenSource;
+        private bool _showNoData;
+        private bool _showList;
+
+        public SendsPageViewModel()
+        {
+            _searchService = ServiceContainer.Resolve<ISearchService>("searchService");
+            Sends = new ExtendedObservableCollection<SendView>();
+            SendOptionsCommand = new Command<SendView>(SendOptionsAsync);
+        }
+
+        public Command SendOptionsCommand { get; set; }
+        public ExtendedObservableCollection<SendView> Sends { get; set; }
+        public Func<SendView, bool> Filter { get; set; }
+
+        public bool ShowNoData
+        {
+            get => _showNoData;
+            set => SetProperty(ref _showNoData, value, additionalPropertyNames: new string[]
```

Comment:
`new string[]` can be simplified to `new []`

Reply:
done, and...